R/create_action.R

```diff
@@ -2,7 +2,6 @@
 #' @param action_type the type of action
 #' @export
 create_action <- function(action_type) {
-  # TODO: check if action_type is a valid action type
```

Comment:
I would have liked to keep the comment, but checking the action type will probably happen in the future anyways :+1: